example/rds.tf

```diff
@@ -50,30 +50,63 @@ module "example_team_rds" {
   # db_parameter = [
   #   {
   #     name         = "rds.force_ssl"
-  #     value        = "true"
+  #     value        = "0"
   #     apply_method = "pending-reboot"
   #   }
   # ]
 
-  # Set below values if you want to create read replica db instance
+  # use "allow_major_version_upgrade" when upgrading the major version of an engine
+  allow_major_version_upgrade = "true"
 
-  # Set the database_name of the source db
-  # db_name = module.example_team_rds.database_name
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = aws.london
+  }
+}
 
-  # If specifies, this resource is a Replicate database. Set the db_identifier of the source db 
-  # replicate_source_db         = module.example_team_rds.db_identifier
+# To create a read replica, use the below code and update the values to reflect your rds instance
```

Comment:
Could you clarify "reflect your rds instance"?

I think it would help to say something like this:

> update the values to specify the RDS instance from which you are replicating. In  this example, we're assuming that `example_team_rds` is the source RDS instance,and `example-team-read-replica` is the replica we are creating.